conf/ucd_sonic.config

```diff
@@ -0,0 +1,38 @@
+//UCD_SONIC config
+params {
+  config_profile_name = 'UCD_SONIC'
+  config_profile_description = 'University College Dublin Sonic HPC profile provided by nf-core/configs.'
+  config_profile_contact = 'Bruce Moran (@brucemoran)'
+  config_profile_url = 'https://www.ucd.ie/itservices/ourservices/researchit/researchcomputing/sonichpc/'
+  //max_memory = 1.MB //yes really, allocation made based on other factors and cannot be done via params.max_memory...
+  max_cpus = 40
+  max_time = 12.h
+}
+
+process {
+  executor = 'slurm'
+  queue = 'shared'
+  queueSize = 50
+  submitRateLimit = '10 sec'
+  maxRetries = 2
+  beforeScript = 'export NXF_OPTS="-Xms2G -Xmx40G"; module load nextflow/22.04.5.5708 singularity/3.5.2'
+  clusterOptions = { "--mem 1M" } 
+  cache = 'lenient'
+  memory = 1.MB
+}
+
+cleanup = true
+
+profiles {
+  moranb {
```

Comment:
What is this exactly? Is it for a specific lab? This should be documented

Reply:
Removed, yes it was for a specific lab.